codalab/lib/bundle_cli.py

```diff
@@ -3184,18 +3198,28 @@ def do_wls_command(self, args):
             'worksheets',
             params={'keywords': args.keywords, 'include': ['owner', 'group_permissions']},
         )
+
         if args.uuid_only:
             for row in worksheet_dicts:
                 print(row['uuid'], file=self.stdout)
         else:
             if worksheet_dicts:
+                if len(worksheet_dicts) == bundle_model.SEARCH_RESULTS_LIMIT:
```

Comment:
Could you factor this block out to avoid code duplication?

Reply:
Sounds good.